Sources/KlaviyoSwift/InternalAPIModels.swift

```diff
@@ -227,10 +227,34 @@ extension Klaviyo.Profile.Attributes.Location: Codable {
 
 // MARK: Legacy request data
 
+struct LegacyIdentifiers {
+    let email: String?
+    let phoneNumber: String?
+    let externalId: String?
```

Comment:
Should at least one of the property in the struct be non-nil/required?

Reply:
Nope! We support anonymous identification via a uuid. The old sdk used idfv - identifierForVendor. This PR is about moving away from that one.